cms/templates/cms/home_page.html

```diff
@@ -68,180 +74,204 @@
+              who share your interests. You may start any course for free and then upgrade to the certificate track.
+            </p>
           </div>
-          <div class="enroll-reason col-md-3">
+          <div class="hiw col-md-3">
+            <h4>Demonstrate your new skills</h4>
             <div class="reasons-images-wrap">
-              <img src="{% static 'images/blended.png' %}" alt="">
+              <img src="{% static 'images/hiw2.png' %}" alt="">
             </div>
-
-            <h4>Accelerate Your Master’s</h4>
-            <p>A MicroMasters program is a faster path to a master’s degree from MIT and other universities, and is recognized by employers for providing deep learning in specific career fields.</p>
-
+            <p>Pass digital proctored exams—either a single final exam, or one exam for each course, depending on 
+              the program.</p>
           </div>
-          <div class="enroll-reason col-md-3">
+          <div class="hiw col-md-3">
+            <h4>Earn an invaluable credential</h4>
             <div class="reasons-images-wrap">
-              <img src="{% static 'images/cost.png' %}" alt="">
+              <img src="{% static 'images/hiw3.png' %}" alt="">
             </div>
-             <h4>Save Money &amp; Time</h4>
-            <p>A MicroMasters program is an affordable option for professionals who can’t take off time to earn a traditional master’s degree.</p>
+            <p>A MicroMasters program credential proves that you’ve gained the knowledge and skills to stand out 
+              in your field. This is a flexible option for professionals who can’t take off time to earn a 
+              traditional master’s degree.
+            </p>
           </div>
-          <div class="enroll-reason col-md-3">
+          <div class="hiw col-md-3">
+            <h4>Pursue a Master’s degree</h4>
             <div class="reasons-images-wrap">
-              <img src="{% static 'images/globe-icon.png' %}" alt="">
+              <img src="{% static 'images/hiw4.png' %}" alt="">
             </div>
-             <h4>Grow Your Network</h4>
-            <p>After you earn your MicroMasters® program credential, you will become an affiliate of the MIT Alumni Association.{% if benefits_url %} <a href="{{ benefits_url }}">Learn More</a>{% endif %}</p>
+            <p>Learners who earn the MicroMasters program credential can apply for the Master’s degree program at
+               MIT and other participating universities globally.</p>
+          </div>
+        </div>
+      </section>
+
+      <section class="faqs">
+        <div class="faq-holder">
+          <img src="{% static 'images/faq.png' %}" alt="FAQs">
+          <div class="info">
+            <span class="title">frequently asked questions</span>
+            <span class="description">
+              <a href="#">Find Answers</a> to your questions
```

Comment:
This will link to the ZenDesk help center, once #4734 is finished.

```suggestion
              <a href="https://mitx-micromasters.zendesk.com/hc/en-us">Find Answers</a> to your questions
```